setup.py

```diff
@@ -33,6 +33,7 @@ def get_version():
         'click',
         'finufftpy',
         'importlib_resources>=1.0.2',
+	'imageio',
```

Comment:
indentation?

Reply:
Thanks. I must use different editor for making the change. Removed the need of imageio in the new commit 10cbb7a.

Reply:
Shouldn't you introduce `pillow` in setup.py now that you're importing `PIL`? Don't rely on any other libraries automatically doing that for you.

Reply:
I checked the old one in picking.py. It seems that PIL can be used without explicit pip installation.

Reply:
That's what I'm saying .. it can be used without an explicit pip installation, but if our code is importing `PIL` then it's **our** responsibility to list it in `setup.py`. If scipy or jupyter or whatever stopped using PIL, this code would be hosed. Try it yourself - `pip uninstall pillow` followed by `import PIL`

Reply:
Added `pillow` explicitly to the list of setup.py in new commit e22fff0. It looks that the `conda package` automatically includes pillow previously since the CI passed.